slidechain/export.go

```diff
@@ -25,10 +26,74 @@ import (
 	"github.com/stellar/go/xdr"
 )
 
-const baseFee = 100
+type pegOut struct {
+	Txid     []byte      `json:"txid,omitempty"`
+	AssetXDR string      `json:"asset"`
```

Comment:
Let's stay consistent about XDR-named variables having `[]byte` type and _not_ being base64-encoded.